src/index.html

```diff
@@ -5,8 +5,462 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+      src="./images/logo.png"
+      alt="MOYO logo"
+      class="header__logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a
+              class="nav__link nav__link--is-active is-active"
+              href="#">
```

Comment:
якщо переносих атрибути то закриваючу чкобку потрібно писати на рівні з відкриваючою, переглянь код і пофікси це
```suggestion
            <a
              class="nav__link nav__link--is-active is-active"
              href="#"
            >
```